specification/communication/data-plane/Common/stable/2022-07-13/common.json

```diff
@@ -84,6 +84,49 @@
         "modelAsString": true
       }
     },
+    "CallingCommunicationIdentifierModel": {
```

Comment:
Why are we adding these models to common.json? Common.json is only used for shared models across ACS REST APIs.

Reply:
@DominikMe I added it to the Common.js cause these models can be used again for a different acs events in the future.

Reply:
Let's not add them here. You would also need to create a new api-version for common and can't just modify.